scipy/stats/stats.py

```diff
@@ -5505,6 +5505,12 @@ def ttest_1samp(a, popmean, axis=0, nan_policy='propagate',
     pvalue : float or array
         Two-sided p-value.
 
+    Notes
+    -----
+    The null hypothesis is determined as ``numpy.mean(a, axis) - popmean = 0``.
```

Comment:
These look good. One good (but not necessary) change would be use `np` instead of the full namespace `numpy` as `numpy` is imported as `np` everywhere in SciPy:

```suggestion
    The null hypothesis is determined as ``np.mean(a, axis) - popmean = 0``.
```